### PR TITLE
fix(docs): Fix broken links. Closes #2517

### LINKED ISSUE #2517: [Documentation] Bad link
Broken link from: https://bootstrap-vue.js.org/docs/components/button-group/
to: https://bootstrap-vue.js.org/docs/components/button-group/button-toolbar

### PR DESCRIPTION
**Pull request by jackmu95**
## Description of Pull Request:

This PR fixes some broken links in the docs.

Closes #2517.

## PR checklist:

**What kind of change does this PR introduce?** (check at least one)

- [x] Bugfix
- [ ] Feature
- [ ] Enhancement to an existing feature
- [ ] ARIA accessibility
- [x] Documentation update
- [ ] Other, please describe:

**Does this PR introduce a breaking change?** (check one)

- [ ] Yes
- [x] No

If yes, please describe the impact:

**The PR fulfills these requirements:**

- [x] It's submitted to the `dev` branch, **not** the `master` branch
- [x] When resolving a specific issue, it's referenced in the PR's title (i.e.
      `fixes #xxxx[,#xxxx]`, where "xxxx" is the issue number)
- [x] The PR should address only one issue or feature. If adding multiple features or fixing a bug
      and adding a new feature, break them into separate PRs if at all possible.
- [x] PR titles should following the
      [**Conventional Commits**](https://www.conventionalcommits.org/) naming convention (i.e.
      "fix(alert): not alerting during SSR render", "docs(badge): Updated pill examples, fix typos",
      "chore: fix typo in docs", etc). **This is very important, as the `CHANGELOG` is generated
      from these messages.**

**If new features/enhancement/fixes are added or changed:**

- [ ] Includes documentation updates (including updating the component's `package.json` for slot and
      event changes)
- [ ] New/updated tests are included and passing (if required)
- [ ] Existing test suites are passing
- [ ] The changes have not impacted the functionality of other components or directives
- [ ] ARIA Accessibility has been taken into consideration (does it affect screen reader users or
      keyboard only users? clickable items should be in the tab index, etc)

**If adding a new feature, or changing the functionality of an existing feature, the PR's
description above includes:**

- [ ] A convincing reason for adding this feature (to avoid wasting your time, it's best to open a
      suggestion issue first and wait for approval before working on it)

### CODE REVIEW

File: src/components/button-group/README.md [hunk 1]
```diff
@@ -74,8 +74,8 @@ prop. Split button dropdowns are not supported here.
 
 ## Dropdown menu support
 
-Add [`<b-dropdown>`](./dropdown) menus directly inside your `<b-button-group>`. Note that split
-dropdown menus are not supported when prop `vertical` is set.
+Add [`<b-dropdown>`](docs/components/dropdown) menus directly inside your `<b-button-group>`. Note
```
Comment (tmorehouse) [inline src/components/button-group/README.md]: Should we add a `/` in front of `docs/...`?

File: src/components/button-toolbar/README.md [hunk 2]
```diff
@@ -103,7 +103,7 @@ it is not possble to use key presses to jump out of a text (or test-like) inputs
 
 ## See Also
 
-- [`<b-button-group>`](./button-group)
-- [`<b-dropdown>`](./dropdown)
+- [`<b-button-group>`](docs/components/button-group)
+- [`<b-dropdown>`](docs/components/dropdown)
```
Comment (tmorehouse) [inline src/components/button-toolbar/README.md]: Should we add a `/` in front of `docs/...`?

File: src/components/form-radio/README.md [hunk 3]
```diff
@@ -180,8 +180,8 @@ render them inline.
 
 Render radios with the look of buttons by setting the prop `buttons` to `true` on
 `<b-form-radio-group>`. Set the button variant by setting the `button-variant` prop to one of the
-standard Bootstrap button variants (see [`<b-button>`](./button) for supported variants). The
-default `button-variant` is `secondary`.
+standard Bootstrap button variants (see [`<b-button>`](docs/components/button) for supported
```
Comment (tmorehouse) [inline src/components/form-radio/README.md]: And here as well...

File: src/components/nav/README.md [hunk 4]
```diff
@@ -146,16 +146,16 @@ add them (like above) which will produce something like:
...
 
 ## Using in Navbar
 
 Prop `is-nav-bar` has been deprecated and will be removed in a future release.
 
 ## Tabbed local content support
 
-See the [`<b-tabs>`](./tabs) component for creating tabbable panes of local content (not suited for
-navigation).
+See the [`<b-tabs>`](docs/components/tabs) component for creating tabbable panes of local content
```
Comment (tmorehouse) [inline src/components/nav/README.md]: And here...

File: src/components/nav/README.md [hunk 5]
```diff
@@ -146,16 +146,16 @@ add them (like above) which will produce something like:
 </li>
 ```
 
-Refer to [`<b-dropdown>`](../dropdown) for a list of supported sub-components.
+Refer to [`<b-dropdown>`](docs/components/dropdown) for a list of supported sub-components.
```
Comment (tmorehouse) [inline src/components/nav/README.md]: and here

**Comment by jackmu95** [comment]
@tmorehouse Good catch and all done.

**Pull request merged**